app/views/hyrax/content_blocks/_form.html.erb

```diff
@@ -1,133 +1,145 @@
+    <li id="home-text-nav-item" class="nav-item">
+      <a href="#home_text" role="tab" data-toggle="tab" class="nav-safety-confirm">
+        <%= t(:'hyrax.content_blocks.tabs.home_text') %>
+      </a>
     </li>
-    <li>
-      <a href="#researcher" role="tab" data-toggle="tab" class="nav-safety-confirm"><%= t(:'hyrax.content_blocks.tabs.featured_researcher') %></a>
+    <li id="researcher-nav-item" class="nav-item">
+      <a href="#researcher" role="tab" data-toggle="tab" class="nav-link nav-safety-confirm">
+        <%= t(:'hyrax.content_blocks.tabs.featured_researcher') %>
+      </a>
     </li>
-    <li>
-      <a href="#homepage_about_section_heading" role="tab" data-toggle="tab" class="nav-safety-confirm"><%= t(:'hyrax.content_blocks.tabs.homepage_about_section_heading') %></a>
+    <li id="homepage-about-heading-nav-item" class="nav-item">
+      <a href="#homepage_about_section_heading" role="tab" data-toggle="tab" class="nav-safety-confirm">
+        <%= t(:'hyrax.content_blocks.tabs.homepage_about_section_heading') %>
+      </a>
     </li>
-    <li>
-      <a href="#homepage_about_section_content" role="tab" data-toggle="tab" class="nav-safety-confirm"><%= t(:'hyrax.content_blocks.tabs.homepage_about_section_content') %></a>
+    <li id="homepage-about-content-nav-item" class="nav-item">
+      <a href="#homepage_about_section_content" role="tab" data-toggle="tab" class="nav-safety-confirm">
+        <%= t(:'hyrax.content_blocks.tabs.homepage_about_section_content') %>
+      </a>
     </li>
   </ul>
   <div class="tab-content">
-    <div id="announcement_text" class="tab-pane active">
-      <div class="panel panel-default labels">
+    <div id="announcement_text" class="tab-pane show active">
+      <div class="card labels">
         <%= simple_form_for ContentBlock.for(:announcement), url: hyrax.content_block_path(ContentBlock.for(:announcement)), html: {class: 'nav-safety'} do |f| %>
-          <div class="panel-body">
+          <div class="card-body">
             <div class="field form-group">
                 <%= f.label :announcement %><br />
                 <%# the following line was changed from hyrax to give some context for what this context block does %>
                 <p class="content-block-instructions"><%= t(:'hyrax.content_blocks.instructions.announcement_instructions') %></p>
                 <%= f.text_area :announcement, value: f.object.value, class: 'form-control tinymce', rows: 20, cols: 120 %>
             </div>
           </div>
-          <div class="panel-footer">
-            <%= link_to t(:'hyrax.content_blocks.cancel'), hyrax.admin_admin_sets_path, class: 'btn btn-default pull-right' %>
-            <%= f.button :submit, class: 'btn btn-primary pull-right' %>
+          <div class="card-footer d-flex justify-content-end">
+            <%= f.button :submit, class: 'btn btn-primary text-white mr-2' %>
```

Comment:
Reading note; button and link shifted position in the DOM.

Reply:
looks that way, another Hyrax change
https://github.com/samvera/hyrax/blob/double_combo/app/views/hyrax/content_blocks/_form.html.erb#L31